react-forms/package.json

```diff
@@ -31,8 +31,9 @@
   "license": "MIT",
   "bugs": {"url": "https://github.com/GridProtectionAlliance/gpa-gemstone/issues"},
   "homepage": "https://github.com/GridProtectionAlliance/gpa-gemstone#readme",
-  "devDependencies":   {
+  "devDependencies": {
     "@types/jest": "^27.0.0",
+    "@types/lodash": "^4.14.171",
```

Comment:
```suggestion
```

see below for using `helper-functions` package instead

Reply:
Thank you! I thought you might know a way to do it without the lodash dependency.